torch/_inductor/codegen/common.py

```diff
@@ -511,6 +511,139 @@ def remainder(a, b):
     def load_seed(name, offset):
         return ops.load(name, sympy.Integer(offset))
 
+    @classmethod
+    def _initialize_pointwise_overrides(cls, target):
+        assert target in {"triton", "cpp"}, target
+
+        def pointwise_factory(impl):
+            def func(x):
+                return f"{impl}({x})"
+
+            return func
+
+        for funcname, impls in pointwise_overrides_data.items():
+            impl = impls.get(target)
+            if impl is not None:
+                setattr(cls, funcname, staticmethod(pointwise_factory(impl)))
+
+
+pointwise_overrides_data: Dict[str, Dict[str, str]] = dict(
+    airy_ai=dict(
+        cpp="airy_ai_forward",
+        # triton=None,  # libdevice/triton do not provide Airy Ai function
```

Comment:
yes, the logic is not there yet. Functions that don't have libdevice/triton equivalent, can either be implemented as decompositions (e.g. `i0e`, `i1e`, etc but with a risk of having lower accuracy) or as fallbacks when on CUDA device. Currently, all such functions are fallbacks (including CPU cases). The plan is to implement fallback only when triton/cpp equivalent is not available, otherwise use the implementation that described in `pointwise_overrides_data`.

Reply:
> Actually no need to check, as the tests show it's falling back on CPU

Yes, however, it would be preferred if there would be no fallback on CPU.

Reply:
The fallback logic is now implemented via `triton_fallback` kw argument to `make_pointwise`.